src/components/GridLayout.js

```diff
@@ -336,17 +338,37 @@ class GridLayout extends Component {
             )}
             <Ink />
           </div>
-          <div
-            style={{
-              display: "grid",
-              width: "calc(100% - 10px)",
-              gridTemplateColumns: "repeat(auto-fill, minmax(250px, 1fr))",
-              gridGap: 10
-            }}
-          >
-            {tileComponents.map((component, i) => {
-              return <div key={i}>{component}</div>;
-            })}
+          <div>
+            {width > 700
```

Comment:
Dit is een magic number, staat die 700 niet ergens als een constante of zo? Of hebben we een variabele ergens die bepaalt of het een breed scherm is of niet?

Reply:
Dat laatste... < 700 is wanneer we het als mobile/tablet beschouwen

Reply:
Maar die 700 staat nu op vijf plaatsen in de code, alleen in deze file al drie keer. Als we ooit besluiten dat dat 800 moet worden moet dat op al die plaatsen veranderd worden. Logica voor "is dit mobile of niet" moet op 1 plek staan, anders gaat het geheid inconsistent worden in de toekomst.